.circleci/config.yml

```diff
@@ -60,7 +60,7 @@ commands:
       - run:
           name: Run tests
           working_directory: << parameters.directory >>
-          command: bundle exec fastlane test
+          command: bundle exec fastlane scan
```

Comment:
`test` isn't compatible with integration tests, since it specifies a test plan, and that's not available on the integration tests projects